app/src/main/res/drawable/ic_baseline_check_24.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Check for this & other icon: could we maybe depend on a material icon rather than manually defining these?